test/integ/matMul/src/matMul.cpp

```diff
@@ -150,26 +144,20 @@ namespace alpaka
                 TIndex const& n,
                 TIndex const& k,
                 TElem const& alpha,
-                TElem const* const A,
-                TIndex const& lda,
-                TElem const* const B,
-                TIndex const& ldb,
+                alpaka::Accessor<TElem*, TElem, TIndex, 2, alpaka::ReadAccess> const A,
```

Comment:
Switch to `TMemoryHandle` was forgotten here.

Reply:
Thx!